bgc_data/combined_bgc_map.sql

```diff
@@ -5,8 +5,8 @@ CREATE MATERIALIZED VIEW combined_bgc_map AS
     "Project",
     "StationName",
     "TripCode",
-    "SampleTime_Local"::date AS "SampleDate_Local",
-    "SampleTime_UTC"::date AS "SampleDate_UTC",
+    "SampleTime_Local",
```

Comment:
This `SampleTime_Local` will now not match any field of the same name in any of the `_data` layers. Might lead to some confusion, especially if we use this field to filter on in the portal?

Reply:
My suggestion would be to rename this to `TripTime_Local` (and also have `TripTime_UTC`) and include it in all the `_data` views - then we can filter on that.
@clairedavies what do you think?